core/src/main/java/org/helsinki/paytrail/request/common/PaytrailPostRequest.java

```diff
@@ -11,17 +11,20 @@
 import org.helsinki.paytrail.response.PaytrailResponse;
 
 import java.nio.charset.StandardCharsets;
+import java.util.Map;
 
 @Slf4j
-public abstract class PaytrailPostRequest<T extends PaytrailResponse, B extends PaytrailPayload<?>> extends PaytrailRequest<T> {
```

Comment:
Refactored the way payload type defined for the classes that extend this this abstract. Before this change it was not possible to extend this class without providing a Payload - this became a problem for requests that does not need any request body.